src/main/java/com/microsoft/aad/msal4j/TokenCache.java

```diff
@@ -16,12 +18,10 @@
  */
 public class TokenCache implements ITokenCache {
 
-    protected static final int MIN_ACCESS_TOKEN_EXPIRE_IN_SEC = 5*60;
+    static final int MIN_ACCESS_TOKEN_EXPIRE_IN_SEC = 5*60;
+
+    private ReadWriteLock lock = new ReentrantReadWriteLock();
 
-    /**
-     *  Constructor for token cache
-     * @param tokenCacheAccessAspect {@link ITokenCacheAccessAspect}
-     */
     public TokenCache(ITokenCacheAccessAspect tokenCacheAccessAspect) {
```

Comment:
Please include comments for public constructors

Reply:
added